libs/dh/metering-point/feature-overview/src/lib/breadcrumbs/dh-breadcrumbs.component.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Component, Input } from '@angular/core';
+import { dhMeteringPointPath } from '@energinet-datahub/dh/metering-point/routing';
+
+@Component({
+  selector: 'dh-breadcrumbs',
+  templateUrl: './dh-breadcrumbs.component.html',
+  styleUrls: ['./dh-breadcrumbs.component.scss'],
+})
+export class DhBreadcrumbsComponent {
+  meteringPointAbsolutePath = ['/', dhMeteringPointPath];
+
+  @Input() meteringPointId = '';
+}
```

Comment:
issue: let's test the breadcrumb through an integration test for the metering points overview component